docs/src/index.md

```diff
@@ -1,54 +1,44 @@
 # API Documentation
 
-Below is the API documentation for Onda.jl.
-
-For general information regarding the Onda Format itself, please see [beacon-biosignals/OndaFormat](https://github.com/beacon-biosignals/OndaFormat).
-
-For a nice introduction to the package, see the [Onda Tour](https://github.com/beacon-biosignals/Onda.jl/blob/master/examples/tour.jl).
+We highly recommend that newcomers walk through the [Onda Tour](https://github.com/beacon-biosignals/Onda.jl/blob/master/examples/tour.jl) before diving into this reference documentation.
 
 ```@meta
 CurrentModule = Onda
 ```
 
 ## Support For Generic Path-Like Types
 
-Onda.jl attempts to be as agnostic as possible with respect to the storage system
-that sample data, Arrow files, etc. are read from/written to. As such, any path-like
-argument accepted by an Onda.jl API function should generically "work" as long
-as the argument's type supports:
+Onda.jl attempts to be as agnostic as possible with respect to the storage system that sample data, Arrow files, etc. are read from/written to. As such, any path-like argument accepted by an Onda.jl API function should generically "work" as long as the argument's type supports:
 
 - `Base.read(path)::Vector{UInt8}` (return the bytes stored at `path`)
 - `Base.write(path, bytes::Vector{UInt8})` (write `bytes` to the location specified by `path`)
 
-For backends which support direct byte range access (e.g. S3), `Onda.read_byte_range` may
-be overloaded for the backend's corresponding path type to enable further optimizations:
+For backends which support direct byte range access (e.g. S3), `Onda.read_byte_range` may be overloaded for the backend's corresponding path type to enable further optimizations:
 
 ```@docs
 Onda.read_byte_range
 ```
 
-## `*.onda.annotations.arrow`
+## `onda.annotation`
```

Comment:
No real opinion here but out of curiosity why drop the plural?

Reply:
https://beacon-biosignals.github.io/Legolas.jl/stable/schema/#Naming-Conventions-1